src/mono/CMakeLists.txt

```diff
@@ -181,6 +181,12 @@ string(TOLOWER ${CLR_CMAKE_HOST_OS} CLR_CMAKE_HOST_OS)
 # TODO: check if we can enable this on more platforms
 # set(MONO_KEYWORD_THREAD "__thread")
 
+check_function_exists(clock_gettime HAVE_CLOCK_GETTIME)
```

Comment:
This should go to cmake/configure.cmake. A definition of HAVE_CLOCK_GETTIME should be added to cmake/config.h.in.

Reply:
done